src/protocol.cpp

```diff
@@ -47,6 +47,8 @@ const char *WTXIDRELAY="wtxidrelay";
 const char *SENDTXRCNCL="sendtxrcncl";
 } // namespace NetMsgType
 
+const std::string NET_MESSAGE_TYPE_OTHER = "*other*";
```

Comment:
This was moved over from net.cpp. Should it be in the `NetMsgType` namespace? If so, what about the `allNetMessageTypes` array?